### DISCUSSION TITLE
Looking for Robolectric sample application

### DISCUSSION THREAD

**Discussion by hellosagar** [robolectric/robolectric#7111] [General]
I'm looking for a Robolectric sample application that can demonstrate all the Robolectric APIs. In the documentation, it refers to a repository [testing-samples](https://github.com/android/testing-samples) but this contains whole a lot of tests modules.
In which module or repo I should look into exactly?

**Comment by hellosagar** [comment]
Please ignore this is a stupid question, later I found that docs are already good and explained the most API, contributions guidelines, shadows, and found some sample apps for robolectric from other developers and in the articles.

**Comment by utzcoz** [comment]
@hoisie I received some feedback from friends that they need official samples of Robolectric. For experienced developers and contributors, they can learn from Robolectric's internal test suite, but it's difficult to spread Robolectric to their friends or colleagues who don't have experience of Robolectric. I want to know whether we can create some samples of Robolectric, and maintain them with community.

**Comment by utzcoz** [comment]
@hoisie I think it can be a great project for GSoC 2023 if we apply it. We can make https://github.com/robolectric/deckard-gradle233 live again.
